AMaDda/Model/UserNotificationManager.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  UserNotificationManager.swift
+//  AMaDda
+//
+//  Created by Lee Myeonghwan on 2022/07/18.
+//
+
+import UserNotifications
+
+final class UserNotificationManager {
+    private let notificationCenter = UNUserNotificationCenter.current()
+    private let notificationCycleDay = 36
```

Comment:
```suggestion
    private let notificationCycleDay: Int = 36
```

p3;

타입 어노테이션을 적극적으로 사용해보면 어떨까요?
현재는 36이라는 숫자만 봤을 때 Int인지 Double인지 CGFloat인지 컴파일러가 추정하는 시간이 필요할 것으로 보이는데요 👀

타입 어노테이션을 활용하면 타입 추정을 줄일 수 있어 컴파일 성능 향상에 도움이 됩니다!